### DISCUSSION TITLE
Content-Security-Policy changes for Cloud POS

### DISCUSSION THREAD

**Discussion by MSPOSmsyczewski** [microsoft/Dynamics365Commerce.Solutions#197] [Q&A] [answered]
Hello,

Is anybody aware how can we change CSP for Cloud POS? We have an ongoing integration that requires an iframe displayed within POS and we are struggling to get the Store Commerce hybrid installation going due to blocked content.

**Comment by madyke** [comment, marked as answer]
Hi @MSPOSmsyczewski - I believe this was already addressed internally with the CSS team, but just reposting some details here in case other customers have similar questions.

Unfortunately, we do not officially support extensibility for CSP for Cloud POS when Store Commerce is used in Hybrid mode. Our best recommendation available right now is to switch to using In-Store topology for Store Commerce. While some partners found ways to implement this in certain RSSU configurations for Retail SDK, those were not officially supported scenarios and as such, are not guaranteed to still work after the migration to Commerce SDK.

We have considered this feature in the past, but it did not have sufficient support to take it as a feature request at that time. If others believe this would be a valuable feature, they can vote on this feature request in the Ideas portal:
[Microsoft Idea - Content-Security-Policy extensibility](https://experience.dynamics.com/ideas/idea/?ideaid=eac6383c-2d9e-ed11-9ac4-0003ff45c94c#)
